### ISSUE TITLE
Deployment overshadows services under ideacode.com

### ISSUE THREAD

**Issue by bishopb** [ideacode/ideacode.github.io#1]
Having this static site deployed to ideacode.com now overshadows services that used to exist as paths on ideacode.com. For example, `ideacode.com/roundcube` is now a 404.

**Comment by muvohf4ieph8HeeSu8e** [comment]
@bishopb I need to add a page with an auto redirect, via javascript, pointing to https://mail.ideacode.com which is running updated roundcube.  I think the services should be subdomains not folders, even though folders is often easier/more simple.  Also subdomains allow services to move environments, while folders would not.

**Comment by bishopb** [comment]
I favor subdomains as well. Subdomains are for services, paths are for routes within a service.

**Comment by bishopb** [comment]
> I need to add a page with an auto redirect, via javascript, pointing to https://mail.ideacode.com which is running updated roundcube.

Some users rely on password managers to auto-fill their `ideacode.com/roundcube` credentials. Redirecting from `ideacode.com` to `mail.ideacode.com` is a [`same-site` but not `same-origin`](https://web.dev/same-site-same-origin/) movement, and I'm not sure how password managers will react. `same-origin` is the most conservative, and any password managers that follow it will require the users to supply their credentials. `same-site` will match, but I'm not sure if any password managers do this (though this [official article](https://developer.chrome.com/blog/site-affiliation/) says Chrome does).

My point is that we may have support requests popping up even after the redirect, related to password management.

**Comment by muvohf4ieph8HeeSu8e** [comment]
We'll need to add a support page: how to use our email system.  which includes instructions for roundcube, independent of url.  also include directions for using any email client, the smtp and imap settings to use(though not how to configure a specific email client, unless we recommend one and will support using that one).  and instructions for generic password manager usage.  the password manager specifics, like how to update a password in the manager, is beyond our support capabilities.  more specifically, I would say "use a password manager to help keep a strong password and change it routinely" versus giving specific examples of how to use "X" password manager.

**Comment by muvohf4ieph8HeeSu8e** [comment]
does supporting an email service including supporting the password manager being used?  I can make some general statements recommending the use of a password manager.  I can also add some detailed instructions specific to an individual password manager, in my case that would be `pass`.  Not sure my instructions would be helpful except for other `pass` users.  Some documentation on accessing roundcube, and what to use for the email address, may help.  plus I do recommend use of a password manager, to ensure strong passwords.  And more general documentation about imap, smtp, and starttls settings could be added to allow users to use other email clients, though support for a specific email client would be limited, depend on the email client(again I could add some documentation for `neomutt` if that would be helpful which seems doubtful).  if you want to then add documentation for a specific password manager that would be fine, but you'll need to maintain that.

**Comment by bishopb** [comment]
> does supporting an email service including supporting the password manager being used

No, but understanding how our users use our services allows us to predict impact on usability and consequent support burden from changes we make.

In this case, we know that we have older users (65+) who are using older browsers (IE 11) with older, automatic and built-in password management from the OS (Windows 8) who will pick up their (analog) phone and call us.

So things we might do include:

1. On the `ideacode.com/roundcube` page, rather than just redirect, drop an interstitial message that says something like "Access to email on the web has moved to a new location. Please visit our new email location at `https://mail.ideacode.com/`, and make sure to update your bookmarks. If you stored your password in your browser, your browser may not know how to provide your password on the new location. If you need help with your password, please call your ideacode support representative."
1. On the `https://mail.ideacode.com/` page, place a message like "If you are having difficulty accessing your email, please call your ideacode support representative."
1. Review our internal documentation on how to manage people's credentials and ensure it's up to date.

I would not attempt to document how to use Roundcube, or how to use particular password managers, as a response to this incident. However, we may want to consider documenting those as separate tasks.

**Comment by muvohf4ieph8HeeSu8e** [comment]
Updated the text on the ideacode.com/roundcube page.  I'll work on adding the text to the roundcube login screen.

yes, we need more documentation on the little steps.  and some general documentation for our users.  we can fill out the documentation by adding it as needed(when they have a question we can create the page during the answering process)
